Dockerfile

```diff
@@ -6,6 +6,7 @@ RUN apt-get update && apt-get -y install  unzip \
                         bc \
                         git \
                         build-essential \
+                        golang \
```

Comment:
Emm, the current `apt-get install` packages in [Dockerfile](https://github.com/phusl/boot2docker/blob/xentools/Dockerfile) seems not in alphabet sorting.
Should I push a commit in this PR to sort out?